companies/shoop.json

```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "shoop",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
```

Comment:
The categories are deliberately very vague and may be applied generously. Personally, I don't think *finance* fits too well, I would have probably gone with *ads* and *commerce*.

If none of the categories fit, it's also fine not to specify any, by the way.

Reply:
I agree that "ads" and "commerce" fits better.